pkg/asset/targets/targets.go

```diff
@@ -0,0 +1,67 @@
+package targets
+
+import (
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/cluster"
+	"github.com/openshift/installer/pkg/asset/ignition/bootstrap"
+	"github.com/openshift/installer/pkg/asset/ignition/machine"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/kubeconfig"
+	"github.com/openshift/installer/pkg/asset/manifests"
+	"github.com/openshift/installer/pkg/asset/templates/content/bootkube"
+	"github.com/openshift/installer/pkg/asset/templates/content/openshift"
+	"github.com/openshift/installer/pkg/asset/tls"
+)
+
+var (
+	// InstallConfig are the install-config targeted assets.
+	InstallConfig = []asset.WritableAsset{
+		&installconfig.InstallConfig{},
+	}
+
+	// Manifests are the manifests targeted assets.
+	Manifests = []asset.WritableAsset{
+		&manifests.Manifests{},
+		&manifests.Openshift{},
+	}
+
+	// ManifestTemplates are the manifest-templates targeted assets.
+	ManifestTemplates = []asset.WritableAsset{
+		&bootkube.KubeCloudConfig{},
```

Comment:
Is there a way to avoid duplicating this slice?

```console
$ git grep -A3 'Templates = ' 8f8bfcf31651 -- pkg/asset/templates
8f8bfcf31651:pkg/asset/templates/templates.go:var Templates = []asset.WritableAsset{
8f8bfcf31651:pkg/asset/templates/templates.go-	&bootkube.KubeCloudConfig{},
8f8bfcf31651:pkg/asset/templates/templates.go-	&bootkube.MachineConfigServerTLSSecret{},
8f8bfcf31651:pkg/asset/templates/templates.go-	&bootkube.OpenshiftServiceCertSignerSecret{},
$ git grep -A3 'ManifestTemplates = ' 8f8bfcf31651 -- pkg/asset/targets/targets.go
8f8bfcf31651:pkg/asset/targets/targets.go:	ManifestTemplates = []asset.WritableAsset{
8f8bfcf31651:pkg/asset/targets/targets.go-		&bootkube.KubeCloudConfig{},
8f8bfcf31651:pkg/asset/targets/targets.go-		&bootkube.MachineConfigServerTLSSecret{},
8f8bfcf31651:pkg/asset/targets/targets.go-		&bootkube.OpenshiftServiceCertSignerSecret{},
```